release_notes/ocp_3_6_release_notes.adoc

```diff
@@ -0,0 +1,2608 @@
+
+- Larger memory space to enable larger cluster sizes.
+- Increased stability in adding and removing nodes in general life cycle actions.
+- A significant performance boost.
+
+A migration playbook will be provided in the near future allowing
+upgraded environments to migrate to the v3 data model.
+
+[[ocp-36-cluster-wide-control-of-ca]]
+==== Cluster-wide Control of CA
+
+You now have the ability to change the certificate expiration date en mass
+across the cluster for the various framework components that use TLS.
+
+We offer new cluster variables per framework area so that you can use different
+time-frames for different framework components. Once set, issue the new
+`redeploy-openshift-ca` playbook. This playbook only works for redeploying the
+root CA certificate of {product-title}. Once you set the following options, they
+will be effective in a new installation, or they can be used when redeploying
+certificates against an existing cluster.
+
+.New Cluster Variables
+----
+# CA, node and master certificate expiry
+openshift_ca_cert_expire_days=1825
+openshift_node_cert_expire_days=730
+openshift_master_cert_expire_days=730
+
+# Registry certificate expiry
+openshift_hosted_registry_cert_expire_days=730
+
+# Etcd CA, peer, server and client certificate expiry
+etcd_ca_default_days=1825
+----
+
+[[ocp-36-general-stability]]
+==== General Stability
+
+{product-title} engineering and the OpenShift Online operations teams have been
+working closely together to refactor and enhance the installer. The
+{product-title} 3.6 release includes the culmination of those efforts, including:
+
+- Upgrading from {product-title} 3.5 to 3.6
+- Idempotency refactoring of the configuration role
+- Swap handling during installation
+- All BYO playbooks pull from a normalized group source
+- A final port of operation’s Ansible modules
+- A refactoring of excluder roles
+
+[[ocp-36-metrics-and-logging]]
+=== Metrics and Logging
+
+[[ocp-36-removing-metrics-deployer-and-removing-logging-deployer]]
+==== Removing Metrics Deployer and Removing Logging Deployer
+
+The metrics and logging deployers were replaced with `playbook2image` for `oc
+cluster up` so that `openshift-ansible` is used to install logging and metrics:
+
+----
+$ oc cluster up --logging --metrics
+----
+
+Check metrics and logging pod status:
+
+----
+$ oc get pod -n openshift-infra
+$ oc get pod -n logging
+----
+
+[[ocp-36-expose-elasticsearch-as-a-route]]
+====  Expose Elasticsearch as a Route
+
+By default, Elasticsearch deployed with OpenShift aggregated logging is not
+accessible from outside the logging cluster. This enables a route for external
+access to Elasticsearch for those tools that want to access its data.
+
+Direct access to Elasticsearch using only your OpenShift token. You have the
+ability to provide the external Elasticsearch and Elasticsearch Operations host
+names when creating the server certificate (similar to Kibana). Ansible tasks
```

Comment:
Perhaps substitute the first sentence with:

```
Direct access to the Elasticsearch instance is enabled using your OpenShift token.
```

And consider the last sentence changed to:

```
The provided ansible tasks simplify route deployment.
```